install/0000_80_machine-config-operator_05_osimageurl.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
 data:
   releaseVersion: 0.0.1-snapshot
-  # The OS payload, managed by the daemon + pivot + rpm-ostree
-  # https://github.com/openshift/machine-config-operator/issues/183
+  # This (will eventually) replace the below when https://github.com/openshift/enhancements/pull/1032
+  # progresses towards the default.
```

Comment:
To be clear, this value (and the values found in the `install/image-references` file get replaced by the CVO (cluster version operator).